packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -4,225 +4,242 @@ import type { BadgeState } from './Badge.types';
     boxSizing: 'border-box',
     alignItems: 'center',
     justifyContent: 'center',
-    backgroundColor: tokens.colorBrandBackground,
-    ...shorthands.borderColor(tokens.colorBrandBackground),
-    color: tokens.colorNeutralForegroundOnBrand,
     fontWeight: tokens.fontWeightSemibold,
     ...shorthands.borderWidth(tokens.strokeWidthThin),
     ...shorthands.borderStyle('solid'),
     fontFamily: tokens.fontFamilyBase,
     position: 'relative',
   },
-  rootTiny: {
+
+  // size
+
+  tiny: {
     width: '6px',
     height: '6px',
     fontSize: '4px',
   },
-  rootExtraSmall: {
+  'extra-small': {
     width: '10px',
     height: '10px',
     fontSize: '6px',
   },
-  rootSmall: {
+  small: {
     minWidth: '16px',
     height: '16px',
     ...shorthands.padding('2px'),
     ...shorthands.gap('4px'),
     fontSize: '8px',
   },
-  rootMedium: {
+  medium: {
     height: '20px',
     minWidth: '20px',
     ...shorthands.gap('4px'),
     ...shorthands.padding('4px'),
     fontSize: '10px',
   },
-  rootLarge: {
+  large: {
     minWidth: '24px',
     height: '24px',
     ...shorthands.padding('4px'),
     fontSize: '12px',
     ...shorthands.gap('4px'),
   },
-  rootExtraLarge: {
+  'extra-large': {
     minWidth: '32px',
     height: '32px',
     ...shorthands.padding('6px'),
     ...shorthands.gap('6px'),
     fontSize: '12px',
     ...shorthands.borderWidth(tokens.strokeWidthThick),
   },
-  rootRounded: {
+
+  // shape
+
+  square: {
+    // Default border radius
+  },
+  rounded: {
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
   },
-  rootRoundedSmallToTiny: {
+  roundedSmallToTiny: {
     ...shorthands.borderRadius(tokens.borderRadiusSmall),
   },
-  rootCircular: {
+  circular: {
     ...shorthands.borderRadius('99px'),
   },
-  rootGhost: {
-    backgroundColor: 'transparent',
-    ...shorthands.borderStyle('none'),
-    color: tokens.colorBrandBackground,
+
+  // appearance
+
+  ghost: {
+    ...shorthands.borderColor('transparent'),
   },
-  rootOutline: {
-    backgroundColor: 'transparent',
+
+  // appearance-color
+
+  'filled-brand': {
+    backgroundColor: tokens.colorBrandBackground,
+    color: tokens.colorNeutralForegroundOnBrand,
     ...shorthands.borderColor(tokens.colorBrandBackground),
+  },
+  'outline-brand': {
     color: tokens.colorBrandBackground,
   },
-  rootTint: {
+  'tint-brand': {
     backgroundColor: tokens.colorBrandBackground2,
     color: tokens.colorBrandForeground2,
     ...shorthands.borderColor(tokens.colorBrandStroke2),
   },
-  rootFilledDanger: {
+  'ghost-brand': {
+    color: tokens.colorBrandBackground,
+  },
+  'filled-danger': {
     backgroundColor: tokens.colorPaletteRedBackground3,
     color: tokens.colorNeutralForegroundOnBrand,
     ...shorthands.borderColor(tokens.colorPaletteRedBackground3),
   },
-  rootOutlineDanger: {
+  'outline-danger': {
     color: tokens.colorPaletteRedForeground3,
-    ...shorthands.borderColor(tokens.colorPaletteRedForeground3),
```

Comment:
I removed the specification of `borderColor` from the places where it matched `color`, since it defaults to `currentColor`, and this makes the styles a little smaller.